tests/multiply/test.js

```diff
@@ -1,22 +1,32 @@
-const {assert} = require("chai");
+import {readFileSync} from "fs";
 
-const LC = require("../../src/lambda-calculus.js");
+import {assert, config as chaiConfig} from "chai";
+chaiConfig.truncateThreshold = 0;
+
+import * as LC from "../../src/lambda-calculus.js";
 LC.config.purity = "LetRec";
 LC.config.numEncoding = "Church";
 // LC.config.verbosity = "Concise"; // reinstate for production
 
-const {multiply} = LC.compile();
+const solutionText = readFileSync(new URL("./solution.txt", import.meta.url), {encoding: "utf8"});
 
 describe("Multiply",function(){
-  it("example tests",()=>{
-    assert.equal( multiply(7)(7), 49 );
-    assert.equal( multiply(11)(11), 121 );
-  });
-  it("random tests",()=>{
-    const rnd = (m,n=0) => Math.random() * (n-m) + m | 0 ;
-    for ( let i=1; i<=100; i++ ) {
-      const m = rnd(i), n = rnd(i);
-      assert.equal( multiply(m)(n), m*n );
-    }
-  });
-});
\ No newline at end of file
+  // REVIEW Is this intentional? If so, it should be tested that `compile` throws
+  // and remove other test cases.
+  try {
+    const {multiply} = LC.compile(solutionText);
```

Comment:
`LC.compile` throws.

Reply:
This test was written by Johan, and is designed to represent the front-page `Multiply` bug-fixing kata, for which ever language has a translation. The text in `solution.txt` actually represents the *initial* code, rather than the working solution, which is why it throws. It should probably be changed to be the final solution, but should we also make an `initial.txt` file to save how the solution setup should look?